### PR TITLE
Update main.json

### PR DESCRIPTION
**Pull request by luveluen**
view/options/index.jade
view/options/profile.jade
view/options/settings.jade

-> locales/en/main.json

my user id is be0d25d1-9e71-400d-9d78-7c06647f419f
and actually all I ever wanted to do was translate ! ^^
If my work is of use I will continue.

### CODE REVIEW

File: locales/en/main.json [hunk 1]
```diff
@@ -516,7 +516,102 @@
     "anHourAgo": "An hour ago",
     "hoursAgo": "hours ago",
     "yesterday":"Yesterday",
-    "daysAgo":"days ago"
+    "daysAgo":"days ago",
+    
+    
+"_commentoptionsindex":"OPTIONSINDEX"
```
Comment (paglias) [inline locales/en/main.json]: this and the following string miss a ending comma

File: locales/en/main.json [hunk 2]
```diff
@@ -516,7 +516,102 @@
...
+"allocateintpop" : "Add a point to Intelligence",
+"displayname" : "Display Name",
+"displayphoto" : "Photo",
+"displayblurb" : "Blurb",
+"photourl" : "Photo Url",
+"statsach" : "Stats & Achievements",
+
+
+
+"_commentoptionssettings":"OPTIONSSETTINGS",
```
Comment (paglias) [inline locales/en/main.json:580]: this introductory strings `commentoptions....` are not necessary, you can remove them (but if you want to keep them in order to recognize different parts of the file than it's ok)

Comment (luveluen) [inline locales/en/main.json:580]: I used them to indicate what .jade file the strings are referring to. I think that might come in handy when someone is going to split the file.

Comment (paglias) [inline locales/en/main.json:580]: :+1:

File: locales/en/main.json [hunk 3]
```diff
@@ -516,7 +516,102 @@
...
+"evenallocation" : "Distribute attribute points evenly&nbsp;",
+"evenallocationpopover" : "Assigns the same number of points to each attribute.",
+"classallocation" : "Distribute attribute points based on Class&nbsp;",
+"classallocationpopover" : "Assigns more points to the attributes important to your Class.",
+"taskallocation" : "Distribute attribute points based on task activity&nbsp;",
+"taskallocationpopover" : "Assigns points based on the Physical (STR), Mental (INT), Social (CON), and Other (PER) categories associated with the tasks you complete.",
+"allocatestr" : "Points allocated to STR:",
+"allocatestrpop" : "Add a point to Strength",
+"allocatecon" : "Points allocated to CON:",
+"allocateconpop" : "Add a point to Condition",
```
Comment (SabreCat) [inline locales/en/main.json]: Should be "Constitution" not "Condition", right?

Comment (luveluen) [inline locales/en/main.json]: my languages played a trick on me -_-

**Comment by paglias** [comment]
Fantastic @luveluen , I'm going to merge this now and the other one on habitrpg repo as soon as it's corrected!

@lefnire can we give him a contrib level? the uuid is in the first pull request description

**Comment by SabreCat** [comment]
@paglias I'm on the contrib level, be just a sec!

**Comment by SabreCat** [comment]
Awarded!

**Comment by luveluen** [comment]
Thank you very much, but wait till you see my translations one day.
There I actually know what I am doing.

**Comment by lefnire** [comment]
@paglias sorry man, don't know why you weren't a GM. You can update users now at https://habitrpg.com/#/options/groups/hall/heroes

**Pull request merged**